### ISSUE TITLE
Overwrite hyper params when resume training

### ISSUE THREAD

**Issue by shane-huang** [intel-analytics/BigDL#347]
In resume training, user needs to provide model and optim state (--model and --state ) of a snapshot for BigDL to resume the training from. It's often the case that user needs to adjust hyper-parameters in optim state (e.g. learning rate schedule, weight decay) in the resumed training. But in current implementations (as in googlenet example), overwriting hyper params in state file is not convenient. Should allow command line configurations to overwrite the configs in state.

**Comment by jason-dai** [comment]
Is this duplicate with #354?

**Comment by shane-huang** [comment]
it's fixed in latest master (0.2.0-SNAPSHOT)
